### DISCUSSION TITLE
Closing FreeTube with the mouse

### DISCUSSION THREAD

**Discussion by davitof** [FreeTubeApp/FreeTube#4533] [General]
There are two ways to open a new FreeTube window with the mouse, but I am unable to find a way to close any FreeTube window with the mouse. And since FreeTube is in full screen mode (which hides the Windows task bar), we can't even close it from the task bar.

Am I missing something, or should I request a new feature?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
FT in Fullscreen is a bug or you did that yourself and you like it. Anyway press F11 to go out fullscreen.

Related #3220

**Comment by davitof** [comment]
I should have thought of it! Yes, I probably activated it long ago and forgot I did. Thank you. 

Which triggers a new question : is there a way to get out of full screen with the mouse only?

This is for if I ever switch back to full screen and then forget again...
